src/mlpack/methods/linear_svm/linear_svm.hpp

```diff
@@ -372,9 +328,9 @@ class LinearSVM
                  const arma::Row<size_t>& labels,
                  const size_t numClasses,
                  OptimizerType optimizer,
-                 const double lambda,
-                 const double delta,
-                 const bool fitIntercept,
+                 const std::optional<double> lambda = std::nullopt,
```

Comment:
```suggestion
                 const double lambda,
```
The same applies here.